varcode/core_logic.py

```diff
@@ -381,6 +383,78 @@ def infer_exonic_effect(variant, transcript):
         variant,
         transcript)
 
+def find_nearest_exon(start, end, exons):
+    """
+    Finds nearest exon to an intronic variant, returns the
+    distance to that exon, along with the Exon object itself.
+    """
+    best_distance = float("inf")
+    best_exon = None
+    for exon in exons:
```

Comment:
i'd consider putting this logic in a distance_between_intervals(start1, end1, start2, end2) function